test/reducer_test.dart

```diff
@@ -7564,6 +7580,151 @@ main() {
     expect(new_helix0_svg.y, closeTo(helix0_svg.y, 0.001));
     expect(new_helix2_svg.y, closeTo(helix1_svg.y, 0.001));
   });
+
```

Comment:
Can we put new tests in their own file? In this case, one specific to testing Undo-Redo functionality (or in such a test file if it already exists).

Reply:
https://github.com/UC-Davis-molecular-computing/scadnano/pull/752/commits/99c69fd512c9f53a26d5fe40d7b7bbe764b6daab